changelog.d/29.feature.md

```diff
@@ -0,0 +1,12 @@
+added `JWT_AUTH_COOKIE_*` settings paralleling Django's
+`SESSION_COOKIE_*` which are used for `JWT_AUTH_COOKIE` and
+`JWT_IMPERSONATION_COOKIE`
+
+This changes the default `Secure` attribute from `False` to
+`True`. Users wishing to use JWT cookies over http (as in no TLS/SSL)
+need to set `JWT_AUTH_COOKIE_SECURE` to `False.`
+
```

Comment:
Would you mind adding this paragraph to the documentation (`### JWT_AUTH_COOKIE_SECURE`) as well?

Reply:
done